functests/storagecluster_creation_test.go

```diff
@@ -0,0 +1,159 @@
+						Spec: k8sv1.PersistentVolumeClaimSpec{
+							StorageClassName: &storageClassName,
+							AccessModes:      []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
+
+							Resources: k8sv1.ResourceRequirements{
+								Requests: k8sv1.ResourceList{
+									"storage": monQuantity,
+								},
+							},
+						},
+					},
+					// Setting empty ResourceLists to prevent ocs-operator from setting the
+					// default resource requirements
+					Resources: map[string]corev1.ResourceRequirements{
+						"mon": corev1.ResourceRequirements{
+							Requests: corev1.ResourceList{},
+							Limits:   corev1.ResourceList{},
+						},
+						"mds": corev1.ResourceRequirements{
+							Requests: corev1.ResourceList{},
+							Limits:   corev1.ResourceList{},
+						},
+						"rgw": corev1.ResourceRequirements{
+							Requests: corev1.ResourceList{},
+							Limits:   corev1.ResourceList{},
+						},
+						"mgr": corev1.ResourceRequirements{
+							Requests: corev1.ResourceList{},
+							Limits:   corev1.ResourceList{},
+						},
+						"noobaa": corev1.ResourceRequirements{
+							Requests: corev1.ResourceList{},
+							Limits:   corev1.ResourceList{},
+						},
+					},
+					StorageDeviceSets: []ocsv1.StorageDeviceSet{
+						{
+							Name:     "example-deviceset",
+							Count:    3,
+							Portable: true,
+							Resources: corev1.ResourceRequirements{
+								Requests: corev1.ResourceList{},
+								Limits:   corev1.ResourceList{},
+							},
+							DataPVCTemplate: k8sv1.PersistentVolumeClaim{
+								Spec: k8sv1.PersistentVolumeClaimSpec{
+									StorageClassName: &storageClassName,
+									AccessModes:      []k8sv1.PersistentVolumeAccessMode{k8sv1.ReadWriteOnce},
+									VolumeMode:       &blockVolumeMode,
+
+									Resources: k8sv1.ResourceRequirements{
+										Requests: k8sv1.ResourceList{
+											"storage": dataQuantity,
+										},
+									},
+								},
+							},
+						},
+					},
+				},
+			}
+			storageCluster.SetGroupVersionKind(schema.GroupVersionKind{Group: ocsv1.SchemeGroupVersion.Group, Kind: "StorageCluster", Version: ocsv1.SchemeGroupVersion.Version})
+		})
+
+		AfterEach(func() {
+			err := ocsClient.Delete().
+				Resource("storageclusters").
+				Namespace(storageCluster.Namespace).
+				Name(storageCluster.Name).
+				VersionedParams(&metav1.GetOptions{}, parameterCodec).
+				Do().
+				Error()
+			Expect(err).To(BeNil())
+		})
+
+		Context("create storagecluster", func() {
+			It("and verify PhaseIgnored status", func() {
+				By("Creating StorageCluster")
+				newSc := &ocsv1.StorageCluster{}
+
+				err := ocsClient.Post().
+					Resource("storageclusters").
+					Namespace(storageCluster.Namespace).
+					Name(storageCluster.Name).
+					Body(storageCluster).
+					Do().
+					Into(newSc)
+				Expect(err).To(BeNil())
+
+				By("Verifying StorageCluster is PhaseIgnored")
+				sc := &ocsv1.StorageCluster{}
+
+				err = ocsClient.Get().
```

Comment:
There's a race here. You'll need to put this in a short "Eventually" loop. Otherwise it's possible our Get might occur before the reconcile loop processes the new StorageCluster object. 

an eventually loop with a 10 second timeout would be fine. Take a look at the `functests/common.go` `WaitForPVCBound()` function. It's super simple.